client/asset/btc/electrum_client.go

```diff
@@ -1111,179 +1111,6 @@ func (ew *electrumWallet) tryRemoveLocalTx(ctx context.Context, txid string) {
-	if err != nil {
-		return nil, err
-	}
-	for _, txIn := range msgTx.TxIn {
-		op := outPoint{txIn.PreviousOutPoint.Hash, txIn.PreviousOutPoint.Index}
-		if _, found := fundingCoins[op]; !found {
-			return nil, fmt.Errorf("prevout %v was not specified but was spent", op)
-		}
-	}
-	var foundOut bool
-	for i, txOut := range msgTx.TxOut {
-		if bytes.Equal(txOut.PkScript, pkScript) {
-			if txOut.Value != int64(send) {
-				return nil, fmt.Errorf("output %v paid %v not %v", i, txOut.Value, send)
-			}
-			foundOut = true
-			break
-		}
-	}
-	if !foundOut {
-		return nil, fmt.Errorf("no output paying to %v was found", address)
-	}
-
-	txid, err := ew.wallet.Broadcast(ctx, txRaw)
-	if err != nil {
-		ew.tryRemoveLocalTx(ew.ctx, msgTx.TxHash().String())
-		return nil, err
-	}
-	return chainhash.NewHashFromStr(txid) // hope this doesn't error because it's already sent
-
-	/* Cannot request change addresses from Electrum!
-
-	change := extra - fees
-	changeAddr, err := ew.changeAddress()
-	if err != nil {
-		return nil, fmt.Errorf("error retrieving change address: %w", err)
-	}
-
-	changeScript, err := txscript.PayToAddrScript(changeAddr)
-	if err != nil {
-		return nil, fmt.Errorf("error generating pubkey script: %w", err)
-	}
-
-	changeOut := wire.NewTxOut(int64(change), changeScript)
-
-	// One last check for dust.
-	if dexbtc.IsDust(changeOut, feeRate) { // TODO: use a customizable isDust function e.g. (*baseWallet).IsDust
-		// Re-calculate fees and change
-		fees = (unfundedTxSize - dexbtc.P2WPKHOutputSize + uint64(inputsSize)) * feeRate
-		send = sum - fees
-	} else {
-		tx.AddTxOut(changeOut)
-	}
-
-	wireOP := wire.NewTxOut(int64(send), pkScript)
-	tx.AddTxOut(wireOP)
-
-	tx, err = ew.signTx(tx)
-	if err != nil {
-		return nil, fmt.Errorf("signing error: %w", err)
-	}
-
-	return ew.sendRawTransaction(tx)
-	*/
-}
-
-// part of the btc.Wallet interface
-func (ew *electrumWallet) sendToAddress(address string, value, feeRate uint64, subtract bool) (*chainhash.Hash, error) {
-	if subtract {
-		return ew.sendWithSubtract(ew.ctx, address, value, feeRate)
-	}
-
-	txRaw, err := ew.wallet.PayTo(ew.ctx, ew.walletPass(), address, toBTC(value), float64(feeRate))
-	if err != nil {
-		return nil, err
-	}
-	msgTx, err := ew.deserializeTx(txRaw)
-	if err != nil {
-		return nil, err
-	}
-	txid, err := ew.wallet.Broadcast(ew.ctx, txRaw)
-	if err != nil {
-		ew.tryRemoveLocalTx(ew.ctx, msgTx.TxHash().String())
-		return nil, err
-	}
-	return chainhash.NewHashFromStr(txid)
-}
-
-func (ew *electrumWallet) sweep(ctx context.Context, address string, feeRate uint64) ([]byte, error) {
-	txRaw, err := ew.wallet.Sweep(ctx, ew.walletPass(), address, float64(feeRate))
-	if err != nil {
-		return nil, err
-	}
-	msgTx, err := ew.deserializeTx(txRaw)
-	if err != nil {
-		return nil, err
-	}
-	_, err = ew.wallet.Broadcast(ctx, txRaw)
-	if err != nil {
-		ew.tryRemoveLocalTx(ctx, msgTx.TxHash().String())
-		return nil, err
-	}
-
-	return txRaw, nil
-}
-
```

Comment:
Looks like we can also remove three of the methods form the `electrumWalletClient interface`, but I kinda wanna keep the `WalletClient` methods in wallet_methods.go (it's the separate `btc/electrum` package, so it's more complete with those left).  You've not removed them, just noting that they'll look unused when the interface in this file is cleaned up.